node-backend/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "node-backend",
+  "version": "1.0.0",
+  "description": "Node runtime environment for frontend test",
```

Comment:
Could you change this to "Node runtime environment for frontend coding challenge"? :prettyplease: